### PR TITLE
Set upper bound on EF Core dependency

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.csproj [hunk 1]
```diff
@@ -51,5 +52,29 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  
+  <Target Name="SetNuspecProperties" BeforeTargets="GenerateNuspec">
+    <PropertyGroup>
+      <EFCoreSqlServerVersion>@(PackageReference->WithMetadataValue('Identity', 'Microsoft.EntityFrameworkCore.SqlServer')->Metadata('Version'))</EFCoreSqlServerVersion>
```
Comment (bricelam) [inline EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.csproj:58]: Bet you didn't know that was possible in MSBuild 😉

**Review by smitpatel — APPROVED** [review]

File: EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.nuspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <projectUrl>$projectUrl$</projectUrl>
+    <description>$description$</description>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <copyright>$copyright$</copyright>
+    <tags>$tags$</tags>
+    <repository type="$repositoryType$" url="$repositoryUrl$" />
+    <dependencies>
+      <group targetFramework=".NETStandard2.0">
+        <dependency id="EntityFrameworkCore.SqlServer.HierarchyId.Abstractions" version="$version$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.EntityFrameworkCore.SqlServer" version="[$efcoreSqlServerVersion$,5.0.0)" exclude="Build,Analyzers" />
```
Comment (bricelam) [inline EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.nuspec:19]: Gonna use `,5.0.0-0)` to prevent prereleases of 5.0 too.

Comment (bricelam) [reply EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.nuspec]: Of course NuGet.org rejects it.

> The package manifest contains an invalid Version: '5.0.0-0'

File: EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.csproj (hunk 1 above)
Comment (roji) [reply EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.csproj]: Whoa... wizardry...

**Pull request closed without merging**